test/functional/activities_controller_test.rb

```diff
@@ -29,10 +29,12 @@
 
 require File.expand_path('../../test_helper', __FILE__)
 
-class ActivitiesControllerTest < ActionController::TestCase
+describe ActivitiesController, type: :controller do
   fixtures :all
 
-  def test_project_index
+  render_views
+
+  it 'project_index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.